kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -13,43 +13,12 @@ Use this script to create an Oracle DB service in the default namespace, with th
 The script assumes that either the image, `container-registry.oracle.com/database/enterprise:12.2.0.1-slim`, is available in the Docker repository, or an `ImagePullSecret` is created for `container-registry.oracle.com`. To create a secret for accessing `container-registry.oracle.com`, see the script `create-image-pull-secret.sh`.
 
 ```
-
-$ ./start-db-service.sh -h    
-usage: ./start-db-service.sh -p <nodeport> -i <image> -s <docker-store> [-h]
- -i  Oracle DB Image (optional)
-    (default: container-registry.oracle.com/database/enterprise:12.2.0.1-slim)
-  -p DB Service NodePort (optional)
-    (default: 30011)
-  -s DB Image PullSecret  (optional)
-    (default: docker-store)
-  -h Help
-
-$ ./start-db-service.sh     
-NodePort[30011] ImagePullSecret[docker-store] Image[container-registry.oracle.com/database/enterprise:12.2.0.1-slim]
-deployment.extensions/oracle-db created
-service/oracle-db created
-[oracle-db-54667dfd5f-76sxf] already initialized ..
-Checking Pod READY column for State [1/1]
-Pod [oracle-db-54667dfd5f-76sxf] Status is Ready Iter [1/60]
-NAME                         READY   STATUS    RESTARTS   AGE
-oracle-db-54667dfd5f-76sxf   1/1     Running   0          8s
-NAME                         READY   STATUS    RESTARTS   AGE
-oracle-db-54667dfd5f-76sxf   1/1     Running   0          8s
-NAME         TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)          AGE
-kubernetes   ClusterIP   10.96.0.1      <none>        443/TCP          27d
-oracle-db    NodePort    10.99.58.137   <none>        1521:30011/TCP   9s
-Oracle DB service is RUNNING with NodePort [30011]
-
-```
-
 For creating a Fusion Middleware domain, you can use the database connection string, `oracle-db.default.svc.cluster.local:1521/devpdb.k8s`, as an `rcuDatabaseURL` parameter in the `domain.input.yaml` file.
```

Comment:
Same comment on mentioning that the namespace is configurable.

Reply:
the purpose of the JIRA is to create a Oracle DB service in a custom name space. Modified the description as follow  ...

Use the script samples/scripts/create-oracle-db-service/start-db-service.sh